### PR TITLE
Implements fullscreen on Linux via prepareSettings.

### PR DESCRIPTION
**Pull request by PetrosKataras**
This allows to launch fullscreen on Linux via `prepareSettings`. Runtime fullscreen toggle remains to be implemented.

### CODE REVIEW

File: src/cinder/app/linux/WindowImplLinuxGlfw.cpp [hunk 1]
```diff
@@ -74,7 +75,12 @@ WindowImplLinux::WindowImplLinux( const Window::Format &format, RendererRef shar
     ::glfwWindowHint( GLFW_SAMPLES, options.getMsaa() );
 
 	auto windowSize = format.getSize();
-	mGlfwWindow = ::glfwCreateWindow( windowSize.x, windowSize.y, format.getTitle().c_str(), NULL, NULL );
+	if( mFullScreen ) {
+		mGlfwWindow = ::glfwCreateWindow( windowSize.x, windowSize.y, format.getTitle().c_str(), ::glfwGetPrimaryMonitor(), NULL );
```
Comment (richardeakin) [inline src/cinder/app/linux/WindowImplLinuxGlfw.cpp]: Is there a way to make this so it isn't always the primary monitor, but the monitor / `ci::Display` that is associated with this `Window`?

**Comment by PetrosKataras** [comment]
This should do the trick although I don't currently have a multi-monitor setup at hand to test.

**Comment by andrewfb** [comment]
Thanks for this @PetrosKataras I also threw in a first pass at runtime toggle-able fullscreen at a175b2af6850a930680d31f13a60b475d6cb5525

**Comment by PetrosKataras** [comment]
👍

**Pull request merged**